scripts/sql/0002-add-metadata-cols-to-users.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE public.users 
+    ADD COLUMN IF NOT EXISTS created_at timestamp DEFAULT (now() at time zone 'utc') NOT NULL;
```

Comment:
Nit: Letting the DB server deciding on the default doesn't sit well with me :) Plus, I do not see the real value from doing that. Imo it should solely be encapsulated within the DAL.

Reply:
yeah, I just did this to maintain backwards compatibility with the API, as Postgres will populate the column with the `DEFAULT`